src/App.js

```diff
@@ -1,31 +1,26 @@
 import React, { Fragment, useEffect } from 'react';
-import { withRouter } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 import { Routes } from './Routes';
-import { connect } from 'react-redux';
 import NotificationPortal from '@redhat-cloud-services/frontend-components-notifications/NotificationPortal';
 import './index.css';
 
-const App = (props) => {
+const App = () => {
+  const history = useHistory();
   useEffect(() => {
     insights.chrome.init();
 
     insights.chrome.identifyApp('access-requests');
-    return insights.chrome.on(
-      'APP_NAVIGATION',
-      event => this.props.history.push(`/${event.navId}`)
+    return insights.chrome.on('APP_NAVIGATION', (event) =>
+      history.push(`/${event.navId}`)
     );
   }, []);
 
   return (
     <Fragment>
       <NotificationPortal />
-      <Routes childProps={ props } />
+      <Routes />
     </Fragment>
   );
 };
 
-/**
- * connect: https://github.com/reactjs/react-redux/blob/master/docs/api.md
- *          https://reactjs.org/docs/higher-order-components.html
- */
```

Comment:
If you prefer `useHistory` to `withRouter` it'd be good to change the starter app.

Reply:
Good point. We should go trough the starter app and clean the code there.